magnum_cluster_api/cmd/image_builder.py

```diff
@@ -96,6 +95,8 @@ def main(operating_system, version, image_builder_version):
         "kubernetes_deb_version": f"{version.replace('v', '')}-00",
         "kubernetes_semver": f"{version}",
         "kubernetes_series": f"{kubernetes_series}",
+        # https://github.com/flatcar/Flatcar/issues/823
+        "ansible_user_vars": "oem_id=openstack",
```

Comment:
nice catch here.

Reply:
There is a lot of hacks around this because we're using an ISO image, @okozachenko1203 is going to contribute to `image-builder` an `openstack` image so that it uses OpenStack to deploy and publish directly to Glance.

Once we have that in place, we should switch the image builder to use that instead.  It'll drive down the dependencies  and we can remove this after.